packages/core/src/percy.js

```diff
@@ -262,7 +262,7 @@ export class Percy {
   // Takes one or more snapshots of a page while discovering resources to upload with the resulting
   // snapshots. Once asset discovery has completed for the provided snapshots, the queued task will
   // resolve and an upload task will be queued separately.
-  snapshot(options) {
+  snapshot(options, callback = null) {
```

Comment:
incorrect usage of standard terms. `callbacks` are fundamentally different than promises in how they should be used and what they do. Dont use that word here.  Use something else, maybe just resolve, reject as two args